macosx/Controller.mm

```diff
@@ -359,6 +359,71 @@ + (void)initialize
     }
 }
 
+void onStartQueue(tr_session* session, tr_torrent* tor, void* vself)
+{
+    auto* controller = (__bridge Controller*)(vself);
+    auto const hashstr = @(tr_torrentView(tor).hash_string);
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        if (auto* const torrent = [controller torrentForHash:hashstr]; torrent != nullptr)
```

Comment:
The `torrent != nullptr` check here and below is redundant – sending a message to `nil` is a no-op in objc

Reply:
Removed in fb93b97